cli/command/utils.go

```diff
@@ -75,9 +76,7 @@ func PrettyPrint(i any) string {
 	}
 }
 
-type PromptError error
-
-var ErrPromptTerminated = PromptError(errors.New("prompt terminated"))
+var ErrPromptTerminated = errdefs.Cancelled(errors.New("prompt terminated"))
```

Comment:
I think it's good to have this error variable. We use it in our tests to verify if we really did terminate the prompt. Consumers of the Prompt function can also check against this specific error and it's not standardized as an `errdefs.Cancelled` error which is checked for when the cobra root command `Execute` method returns an error.

Also cannot make this a const since the value of the error is not a const. To make it a const requires a workaround which i think is a bit overkill